### ISSUE TITLE
CodeChef parsing issue

### ISSUE THREAD

**Issue by tgvdinesh** [jmerle/chelper-companion#5]
[Code Chef contest](https://www.codechef.com/MAY18B/problems/RD19)

Hi,
CodeChef website parsing is not working.

**Comment by jmerle** [comment]
Fixed in 1.2.2.

**Comment by jmerle** [comment]
There is a line separator there, but it doesn't show because you used inline code tags instead of a code block:

```
kattis
      <div id="contest_time">
        <h2 class="title">CodeChef - May Challenge 2018 Division 2</h2>
      </div>

      <div class="headline-wrapper"><h1>Minimum Deletions</h1></div>

      <div class="text-center"><h1>Minimum Deletions</h1></div>

      
      <table class="sample" summary="sample data">
        <pre>2
2
2 3
2
2 4
</pre>
        <pre>0
-1
</pre>
      </table>
    

      <p><strong>Memory limit: </strong>256 MB</p>
```

**Comment by tgvdinesh** [comment]
Needless to say. If a issue of parser is raised in master branch, the fix has to implemented in **feature/universal** branch as well. :)

**Comment by jmerle** [comment]
The feature/universal branch will not be updated anymore. Instead, I am moving development on the universal version of CHelper Companion over to a new repository (not public yet). CHelper Companion will only receive parser bug fix updates, while all new features will go into the new extension (I got a name for it but don't want to make it public yet to prevent other people from grabbing the name before I did). Then, after CHelper, JHelper and Hightail all support the new format, I'll start the migration for everyone to go from CHelper Companion to the new extension.
